gulpfile.babel.js

```diff
@@ -13,6 +13,7 @@ const buffer = require('vinyl-buffer');
 const eslint = require('gulp-eslint');
 const uglify = require('gulp-uglify');
 const mkdirp = require('mkdirp');
+const envify = require('envify/custom');
```

Comment:
I don't expect people to do anything that `js-token` wouldn't handle, but we don't currently have anything enforcing that. There's probably already a ESLint rule for it though.